lib/_internal/file_picker_web.dart

```diff
@@ -12,14 +12,12 @@ class FilePickerWeb extends FilePicker {
 
   final int _readStreamChunkSize = 1000 * 1000; // 1 MB
 
-  static final FilePickerWeb platform = FilePickerWeb._();
```

Comment:
This static is redundant. The static `FilePicker.platform` is preferred, as that is that one that users will be interacting with.